src/ontology/mondo-edit.obo

```diff
@@ -391628,18 +391621,26 @@ subset: nord_rare {source="MONDO:NORD"}
 subset: ordo_disease {source="Orphanet:399329"}
 subset: orphanet_rare {source="Orphanet:399329"}
 subset: rare
+synonym: "epiphysiolysis capitis femoris" RELATED [OMIM:182260]
 synonym: "epiphysiolysis of the upper femur" EXACT [Orphanet:399329]
 synonym: "femoral head epiphysiolysis" EXACT [Orphanet:399329]
 synonym: "SCFE" EXACT ABBREVIATION [Orphanet:399329]
 synonym: "slipped capital femoral epiphysis" EXACT [Orphanet:399329]
+synonym: "slipped femoral capital epiphyses" EXACT [OMIM:182260]
+synonym: "slipped femoral capital epiphyses" EXACT [MONDO:0008431]
```

Comment:
this happens a lot in merges- these synonyms are duplicated. I just delete the duplicate synonym that comes from the other term.

Sometimes the synonym comes from two different sources, and in this case, I would merge them.

In your case, please delete line 391630, thanks!